Airports/EGKA/Basic.txt

```diff
@@ -1,3 +1,3 @@
 Shoreham
 N050.50.08.000 W000.17.50.000
-123.150
\ No newline at end of file
+123.155
```

Comment:
Should be TWR freq?

Reply:
Already set to a TWR frequency. Problem here is, there are currently two TWR frequencies defined for Shoreham (123.155 & 125.405) with the same login code. Suggest removing 125.405 as this seems to be an alternative frequency per AIP?